features/guidance/anticipate-lanes.feature

```diff
@@ -623,40 +623,7 @@ Feature: Turn Lane Guidance
             """
                  /a\
             x – b   d – y
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
+                 \ /
```

Comment:
I increased the grid size above; and with a grid size this high these scenarios no longer model reality.